Src/WitsmlExplorer.Frontend/components/Modals/ReportModal.tsx

```diff
@@ -171,15 +171,14 @@ export const useGetReportOnJobFinished = (jobId: string): BaseReport => {
       NotificationService.Instance.snackbarDispatcherAsEvent.subscribe(
         async (notification) => {
           if (notification.jobId === jobId) {
-            const jobInfo = await JobService.getUserJobInfo(notification.jobId);
-            if (!jobInfo) {
+            const report = await JobService.getReport(jobId);
+            if (!report) {
               setReport(
                 createReport(
                   `The job has finished, but could not find job info for job ${jobId}`
```

Comment:
Change `find job info for` to `find a report for`

Reply:
done